### ISSUE TITLE
Feature Request: Add support for real-time action output

### ISSUE THREAD

**Issue by martezr** [StackStorm-Exchange/stackstorm-terraform#12]
The Terraform pack currently doesn't support the real-time action output streaming. This would be really helpful for Terraform operations such as apply that could take several minutes to complete.

The underlying library supports real-time output (https://github.com/beelit94/python-terraform) and it would be a matter of integrating that with the StackStorm code to properly return the output.

**Comment by kingsleyadam** [comment]
This seems like a pretty straight forward request. Details here: https://github.com/beelit94/python-terraform/blob/master/README.md#terraform-output

The current behavior could be kept with an option to enable real-time output.

**Comment by arm4b** [comment]
Good idea!

This is good first issue, help wanted.
We welcome any takers to contribute this enhancement.

**Comment by kingsleyadam** [comment]
The other issue I see with the current approach is it's grouping stderr and stdout together and returning it as the result.

What's more ideal is if these both bubbled up to stackstorm accordingly so stackstorm will show stderr and stdout as it normally does. And the build can fail or pass purely on the return code (like it does today).

I'm not sure if anyone uses the current result like it is today. I wouldn't think so since it's so hard to parse.

I'll create a pull request adjusting the behavior accordingly.

**Comment by kingsleyadam** [comment]
Added in a PR (https://github.com/StackStorm-Exchange/stackstorm-terraform/pull/24) but it seems stuck. The build initially failed, I added in a new commit to address the issues. But new builds don't seem to be picking up.

**Comment by nmaludy** [comment]
@kingsleyadam GitHub is having an outage this morning, their API is currently half functional that is most likely the issue with the build.

**Comment by kingsleyadam** [comment]
This issue can be closed.
